<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Musicify</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout" stretch="7,3">
    <item>
     <widget class="QWidget" name="leftPanel" native="true">
      <layout class="QVBoxLayout" name="leftLayout">
       <item>
        <widget class="QWidget" name="playerControls" native="true">
         <layout class="QHBoxLayout" name="playerLayout">
          <item>
           <widget class="QPushButton" name="prevButton">
            <property name="text">
             <string>⏮️ Prev</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="skipButton">
            <property name="text">
             <string>⏭️ Skip</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopButton">
            <property name="text">
             <string>⏹️ Stop</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="allSongsTab">
          <attribute name="title">
           <string>All Songs</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QListWidget" name="songListWidget">
             <property name="styleSheet">
              <string notr="true">font-size: 14px;</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="artistTab">
          <attribute name="title">
           <string>Artists</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTreeWidget" name="artistTreeWidget">
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="genreTab">
          <attribute name="title">
           <string>Genres</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QTreeWidget" name="genreTreeWidget">
             <property name="headerHidden">
              <bool>true</bool>
             </property>
             <column>
              <property name="text">
               <string notr="true">1</string>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="actionLayout">
         <item>
          <widget class="QPushButton" name="playSelectedButton">
           <property name="text">
            <string>▶️ Play Selected</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addQueueButton">
           <property name="text">
            <string>➕ Add to Queue</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="rightPanel" native="true">
      <layout class="QVBoxLayout" name="rightLayout">
       <item>
        <widget class="QLabel" name="nowPlayingLabel">
         <property name="styleSheet">
          <string notr="true">font-size: 16px; font-weight: bold; padding: 5px;</string>
         </property>
         <property name="text">
          <string>Now Playing: (Nothing)</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font-size: 14px; font-weight: bold; padding: 5px;</string>
         </property>
         <property name="text">
          <string>Up Next (Queue)</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="queueListWidget">
         <property name="styleSheet">
          <string notr="true">font-size: 13px;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearQueueButton">
         <property name="text">
          <string>Clear Queue</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>